src/Event/SchemaAlterTableAddColumnEventArgs.php

```diff
@@ -21,7 +21,7 @@ class SchemaAlterTableAddColumnEventArgs extends SchemaEventArgs
 
     private AbstractPlatform $platform;
 
-    /** @var array<int, string> */
+    /** @var string[] */
```

Comment:
I don't think that this is the proper fix. The current code expects this to be a `list<string>`. We can call `array_values()` on arrays that we merge into this array, to make sure we don't get string keys.